### PR TITLE
Updates in text and image for release august 2021

### PR DESCRIPTION
**Pull request by karinbredenberg**
The text have been given minor updates to provide some more clarity regarding CEF and the figure text. and the image have been corrected with name for geospatial and new CITS with no name

### CODE REVIEW

File: res/md/common-intro.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-The specifications are designed to help data creators, software developers and digital archives to tackle the challenge of short-, medium- and long-term data management and reuse in a sustainable, authentic, cost-efficient, manageable and interoperable way.
+The specifications are designed to help data creators, software developers, and digital archives to tackle the challenge of short-, medium- and long-term data management and reuse in a sustainable, authentic, cost-efficient, manageable and interoperable way.
 A visualisation of the current specification network can be seen here:
 
 <a name="figi-dip"></a>
 ![OAIS Entities](figs/fig_1_dip.svg "Diagram showing E-ARK specification dependency hierarchy")
 
-**Figure I:** Diagram showing E-ARK specification dependency hierarchy.
+**Figure I:** Diagram showing E-ARK specification dependency hierarchy. Note that the image only shows examples of the available CITS.
```
Comment (carlwilson) [inline res/md/common-intro.md:11]: Note that the image only shows a selection of the published CITS and isn't an exhaustive list.

File: res/md/common-intro.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ The main aim of a Content Information Type Specification (CITS) is to:
 
 - Define, in technical terms, how data and metadata must be formatted and placed within a CSIP Information Package to achieve interoperability in exchanging specific Content Information.
 
-The number of possible Content Information Type Specifications is unlimited. For a list of existing Content Information Type Specifications see the relevant section in the Common Specification for Information Packages.
+The number of possible Content Information Type Specifications is unlimited. For a list of existing Content Information Type Specifications see the DILCIS Board webpage.
```
Comment (carlwilson) [inline res/md/common-intro.md:49]: "DILCIS Board webpage" Should this be a link?

File: res/md/common-intro.md (hunk 1 above)
Comment (karinbredenberg) [reply res/md/common-intro.md]: Updated!

File: res/md/common-intro.md (hunk 2 above)
Comment (karinbredenberg) [reply res/md/common-intro.md]: Have added it, its further down in the document but for clarity.

**Review by carlwilson — APPROVED** [review]

**Pull request merged**